### DISCUSSION TITLE
Welcome to Speechly Discussions!

### DISCUSSION THREAD

**Discussion by langma** [speechly/speechly#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Anandakrishnan66** [comment]
while runnning nextjs app with speechly it' showing "RUNTIME ERROE : application identity invalid".

**Reply by bigdatabaracus** [reply]
Hello @Anandakrishnan66! Your error implies that your appId is somehow wrong.

**Comment by bilal201715** [comment]
Hi, We notice that the mic is not working on ecommerce demo. Can you tell me when it will work and why it is not working?

**Comment by cho-wissem** [comment]
Hello
Creating an account or connecting to the account for access  to api.speechly.com is impossible via gmail or github each time it redirect to this link www.speechly.com/blog/speechly-is- joining-roblox !
Please i need your help

**Comment by superotto** [comment]
Hello,

unfortunately as a result of the acquisition we are not able to sign up new users to the platform any longer. The free tier will be ramped down by 30-Nov-2023. I hope you are able to find good alternatives.

Best regards,

Otto
